tools/mpremote/pyproject.toml

```diff
@@ -1,6 +1,7 @@
 [build-system]
 requires = [
     "setuptools>=42",
-    "wheel"
+    "wheel",
+    "mpy-cross",
```

Comment:
should 'mpy-cross' be pinned to a specific version, or is it intentionally un-versioned ?

Reply:
My intention at the time was to be un-pinned, to always use latest (because I generally always use latest micropython).
However now, there's efforts under way to have the mpy-cross package built / managed as part of this repo and integrate it into mpremote in a cleaner and more compatible way.